fcrepo-http-api/src/main/resources/views/common-register-header.vsl

```diff
@@ -0,0 +1,5 @@
+#if($fedoraProps.getFcrepoRegisterBanner() == "on")
+<div class="alert alert-info" role="alert">
+Fedora is an open source project supported entirely by it's users. To continue the on-going maintenance of the software and ensure Fedora is meeting the needs of the community, we are working to gather information on current installations. We strongly encourage all users to register their instance in the DuraSpace Community Supported Program Registry hosted by LYRASIS. <a class="btn btn-primary" href="https://registry.lyrasis.org/" role="button">Register your installation today</a>
+</div>
```

Comment:
This is just a suggestion, but maybe wrapping the paragraph of text and the button in two separate `<p></p>` tags so they get some space from each other. 
```html
<div class="alert alert-info" role="alert">
<p>Fedora is an open source project supported entirely by it's users. To continue the on-going maintenance of the software and ensure Fedora is meeting the needs of the community, we are working to gather information on current installations. We strongly encourage all users to register their instance in the DuraSpace Community Supported Program Registry hosted by LYRASIS.</p>
<p><a class="btn btn-primary" href="https://registry.lyrasis.org/" role="button">Register your installation today</a></p>
</div>
```